web/components/contract/contract-mention.tsx

```diff
@@ -0,0 +1,54 @@
+import clsx from 'clsx'
+import { Contract } from 'common/contract'
+import { formatMoney } from 'common/util/format'
+import Link from 'next/link'
+import { contractPath, getBinaryProbPercent } from 'web/lib/firebase/contracts'
+import { fromNow } from 'web/lib/util/time'
+import { BinaryContractOutcomeLabel } from '../outcome-label'
+import { getColor } from './quick-bet'
+
+export function ContractMention(props: { contract: Contract }) {
+  const { contract } = props
+  const { outcomeType, resolution } = contract
+  const probTextColor = `text-${getColor(contract)}`
+
+  return (
+    <Link href={contractPath(contract)}>
+      <a
+        className="group inline whitespace-nowrap rounded-sm hover:bg-indigo-50 focus:bg-indigo-50"
+        title={tooltipLabel(contract)}
```

Comment:
Can we use the tippy Tooltip instead of just title? For consistent formatting with all the rest of our tooltips

Reply:
Saw your thoughts `[c] tooltips were too distracting, just made it a title attribute (yes I know this sucks)`

Which is fine, I guess. Actually the best thing would be to show the contract card in a popup on hover, gwern-style (which shouldn't be too hard)